trulens_eval/trulens_eval/pages/Evaluations.py

```diff
@@ -84,12 +85,14 @@
         )
         gb.configure_column('total_tokens', header_name='Total Tokens (#)')
         gb.configure_column('total_cost', header_name='Total Cost (USD)')
+        gb.configure_column('latency', header_name='Latency (Seconds)')
         gb.configure_column('tags', header_name='Tags')
         gb.configure_column('ts', header_name='Time Stamp')
 
-        for feedback_col in evaluations_df.columns.drop(['chain_id', 'ts',
-                                                         'total_tokens',
-                                                         'total_cost']):
+        for feedback_col in evaluations_df.columns.drop(
+            ['chain_id', 'ts', 'total_tokens', 'total_cost', 'record_json',
+             'latency', 'record_id', 'chain_id', 'cost_json', 'chain_json',
+             'input', 'output']):
```

Comment:
can we add a variable name for this list?
something like

non_feedback_fields = ['chain_id', 'ts', 'total_tokens', 'total_cost', 'record_json',
             'latency', 'record_id', 'chain_id', 'cost_json', 'chain_json',
             'input', 'output']
             
evaluations_df.columns.drop(non_feedback_fields)